Sources/Tools/MaxMain/CMakeLists.txt

```diff
@@ -93,7 +93,6 @@ target_link_libraries(MaxMain MaxExport)
 target_link_libraries(MaxMain MaxPlasmaMtls)
 target_link_libraries(MaxMain ${3dsm_LIBRARIES})
 
-target_link_libraries(MaxMain ${EXPAT_LIBRARY})
```

Comment:
There are other references to ${EXPAT_LIBRARY} in plClient, pfPythonTest and plLocalizationEditor

Reply:
I think I already got the one in plClient, and I'll get the other two shortly.